fronts-client/src/types/Config.ts

```diff
@@ -20,14 +20,13 @@ interface Config {
   env: string;
   editions: string[];
   email: string;
-  avararUrl?: string;
+  avatarUrl?: string;
   firstName: string;
   lastName: string;
   sentryPublicDSN: string;
   mediaBaseUrl: string;
   apiBaseUrl: string;
   switches: { [key: string]: boolean };
-  stage: string;
```

Comment:
This doesn't exist in the [server definition](https://github.com/guardian/facia-tool/blob/ba61c4b4086baccfdac42c08a1137344069da63e/app/controllers/DefaultsController.scala#L16) – the relevant value is stored in `env`.